src/wp-includes/html-api/class-wp-html-tag-processor.php

```diff
@@ -2333,7 +2333,6 @@ private function matches() {
 			 * See https://html.spec.whatwg.org/#space-separated-tokens
 			 */
 			while (
-				// phpcs:ignore WordPress.CodeAnalysis.AssignmentInCondition.FoundInWhileCondition
```

Comment:
thanks for the update! this line should disappear from the PR on the next rebase, as the code it's modifying is now gone